README.md

```diff
@@ -1,58 +1,173 @@
+
+|                                     | [Zola](https://getzola.org/)   | [Cobalt](https://cobalt-org.github.io/) | [Hugo](https://gohugo.io/)   | [Pelican](https://blog.getpelican.com/) |
+|:------------------------------------|:------:|:------:|:------:|:-------:|
+| [Netlify](https://www.netlify.com/) |  [![yes]](https://www.getzola.org/documentation/deployment/netlify/)  |  ![no]   |  ![yes]  |   ![no]  |
+| [Vercel](https://vercel.com/)       |  [![yes]](https://www.getzola.org/documentation/deployment/vercel/)  |  ![no]   |  ![yes]  |  ![yes]  |
+| [SourceHut](https://srht.site/)    |  [![yes]](https://www.getzola.org/documentation/deployment/sourcehut/)  |    ?     |     ?    |     ?    |
+| [Github](https://pages.github.com/) |  [![yes]](https://www.getzola.org/documentation/deployment/github-pages/) |   ?   | ![yes] | ![yes] |
+| [Gitlab](https://docs.gitlab.com/ee/user/project/pages/) | [![yes]](https://www.getzola.org/documentation/deployment/gitlab-pages/) | ? | ![yes] | ![yes] |
+
+## Contributing
+
+If you would like to contribute to improve zola together as a community, there are many ways to do so. In this section, we outline a few ways you could help the zola project. If you're looking to make a contribution now, please read [CONTRIBUTING.md](CONTRIBUTING.md).
+
+### Documentation and pro tips
+
+When you notice something isn't clear in the documentation, you may submit a patch to improve it for every one else. If you're doing something which you think could be of interest to other folks, please write about it [on the forums](https://zola.discourse.group/). If you notice some usage patterns emerging on the forums, feel free to turn that into a documentation page to help other people achieve the same goals.
+
+### Usability and architecture
 
-### Supported content formats
+Despite being three decades old, the world wide web isn't always the most convenient platform to use. zola tries to make your life better, but if you're experiencing difficulties with it, don't hesitate to report those. Please be clear about what was confusing or unexpected regarding what you tried to achieve.
 
-- Zola: markdown
-- Cobalt: markdown
-- Hugo: markdown, asciidoc, org-mode
-- Pelican: reStructuredText, markdown, asciidoc, org-mode, whatever-you-want
+zola doesn't shine for all use cases we can think of, and could use feedback in particular for:
 
-### ![ehh] explanations
+- a better [translations system](https://zola.discourse.group/t/rfc-internationalization-system-rework/546)
+- a [unified path](https://github.com/getzola/zola/issues/977) representation
+- support for [non-Markdown input formats](https://zola.discourse.group/t/alternative-input-formats/76)
+- support for [non-HTML output formats](https://zola.discourse.group/t/proposal-custom-output-formats/68/4)
```

Comment:
The last two should be split from the first two to ensure people don't think they have the same level of importance.